src/client/elm/Explorer/View.elm

```diff
@@ -1971,6 +2037,13 @@ viewBaker ctx bakerId addr =
         ]
 
 
+viewCredId : String -> Element Msg
+viewCredId cred = row []
+                      [ html <| Icons.account_key_deployed 18
+                      , text <| String.left 8 cred
+                      ]
```

Comment:
Will it be possible to copy the entire credId?

If somebody wants to look up an account by it that would be useful. It would also make it consistent with other values which are copyable by clicking.

Reply:
@limemloh Could you comment on this since I think you and I were discussing this?

Reply:
The discussion about whether we should have the hover feature is a separate one. I only wanted to be able to copy it.

I agree with you that it is bad design with the hover option, but that is what we have now for other things so if that is the simplest option now we should add it.